pkgs/top-level/all-packages.nix

```diff
@@ -6971,6 +6971,8 @@ in
 
   rescuetime = libsForQt5.callPackage ../applications/misc/rescuetime { };
 
+  resholve = recurseIntoAttrs (callPackage ../development/misc/resholve { });
```

Comment:
Please call the package here with callPackage and move the build support like commented above.

Reply:
changed local